src/Target.h

```diff
@@ -258,6 +258,16 @@ struct Target {
     /** Was libHalide compiled with support for this target? */
     EXPORT bool supported() const;
 
+    /** Given a size (in bytes), return True if the allocation size can fit
```

Comment:
Any suggestions? While it's currently a uniform heuristic across targets, this seemed like a target-specific property.